tgui/packages/tgui/interfaces/VendingSorted.tsx

```diff
@@ -407,8 +412,7 @@ export const VendingSorted = () => {
 
         {isEmpty && (
           <NoticeBox danger className="ItemContainer">
-            Nothing in here seems to be for you. If this is a mistake contact
-            your local administrator.
+            Здесь ничего нет, если это ошибка, свяжитесь с администрацией.
```

Comment:
Кажется, тут не про админов, а про ИЦ администраци?
```suggestion
            Здесь нет ничего, подходящего для так. Если присутствует ошибка, свяжитесь с вашим администратором.
```